### ISSUE TITLE
How to import previously exported csv

### ISSUE THREAD

**Issue by derMart** [eneiluj/cospend-nc#104]
Hi,

thank you very much for your nice work. I am using Cospend App 1.4.3 on Nextcloud 22 instance.
Readme says its possible to export / import projects as csv.
I am able to export a project as csv file, but searching desperately how to import it back.
How / where in the ui is that possible?

best

**Comment by derMart** [comment]
Never mind I found it. You have to go to the Cospend settings which are, a little bit inconspicuous, at the bottom right of the cospend app. Then there is the import possibility. It would be nice, if that path could be documented in the user settings, and also an improvement maybe to put the import next to the create project button, or incorporate it into the project creation experience. Importing is a way of creating a project on the instance. So in essence, I could image project import to be more prominent.

cheers

**Comment by eneiluj** [comment]
I agree importing a project should be more accessible, ideally close to the project creation navigation item.

It's now at the bottom of the navigation sidebar (the left one).
![cosp3](https://user-images.githubusercontent.com/11291457/159169909-9fe8c058-6286-4964-9036-2cfd292d7424.png)


This will be included in the next release which is coming soon.

I would like to add a context menu (three dots) next to the "new project" item but the underlying vue component does not natively allow it. I tried to play with css without success. The menu button is always on top of the "new project" cancel one:

This is fine and nice :grin:
![cosp1](https://user-images.githubusercontent.com/11291457/159169822-73d63219-6f8c-4053-97b6-fe03c4be858a.png)
But I can't get rid of this:
![cosp2](https://user-images.githubusercontent.com/11291457/159169823-37a538cb-a24b-420f-b5e7-57fdfba3f8c6.png)




You can check this in the `import-project-next-to-newProjectNavItem` branch.
